### ISSUE TITLE
Video parameters response has changed

### ISSUE THREAD

**Issue by Xyde** [AlexAplin/nndownload#93]
I've tried using the newest latest version without avail here: https://github.com/AlexAplin/nndownload/issues/92

But this is the log I got when using an older version.

>nndownload_v1_8_1.exe   https://www.nicovideo.jp/user/53610183 -o{uploader_id}" -   "{uploader}\{published}{title}{id}.{ext} -u USERNAME -p  PASSWORD -r 5 -f -a

Returns

>Logging in...
Logged in.
Requesting videos from user 53610183...
1/12
Traceback (most recent call last):
  File "nndownload\nndownload.py", line 1665, in main
  File "nndownload\nndownload.py", line 1595, in process_url_mo
  File "nndownload\nndownload.py", line 926, in request_user
  File "nndownload\nndownload.py", line 883, in request_video
  File "nndownload\nndownload.py", line 1218, in perform_api_request
  File "nndownload\nndownload.py", line 1398, in collect_parameters
KeyError: 'thumbnailURL'

Yesterday I was using v1_8_1 without any issues?

**Comment by Zokhoi** [comment]
Checked and it seems to be due to a schema change on NicoNico's end.

I got it to work with some modifications in [this](https://github.com/Joch2520/nndownload/commit/5f89c7c69cd091ec933b0ccd1ad832587b6b1961) and [this](https://github.com/Joch2520/nndownload/commit/7bb18b50fb70a35759a5b4d26408c12943b3b90c) commit.

I don't know whether there are any more modifications needed as I only fixed lines that spat out errors. I also commented out `thread_id` because I don't know which thread id it was supposed to be.
